### PR TITLE
(webapp): fix tooltip position and reward message

### LINKED ISSUE #474: entity type showing on EOS
Screen Shot 2021-05-10 at 11 41 10

### LINKED ISSUE #704: Wrong Rewards distribution description text
The read more text for rewards distribution mentions partner and non partners  , this on only relevant in lacchain and does not apply to the following networks: 

- mainnet 
- telos
- proton 
- wax 

image

### LINKED ISSUE #708: Geo Map Tooltip position missing

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Resolve #708 
- Resolve #704 
- Resolve #474 

### Steps to test

1. first step
2. next step

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/src/layouts/Dashboard.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Typography from '@material-ui/core/Typography'
 import { isWidthUp } from '@material-ui/core/withWidth'
 import { useTranslation } from 'react-i18next'
 import { useLocation } from 'react-router-dom'
-import Collapse from '@material-ui/core/Collapse'
+// import Collapse from '@material-ui/core/Collapse'
```
Comment (xavier506) [inline webapp/src/layouts/Dashboard.js:13]: @tetogomez  do we need this comment?

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Review by xavier506 — CHANGES_REQUESTED** [review]
please see comments

File: webapp/src/components/GeoMap/CountryMap.js [hunk 2]
```diff
@@ -95,9 +100,13 @@ const ClusterMap = ({ data, map, mapCode }) => {
     <>
       <div ref={myRef} style={{ height: '100vh' }} />
```
Comment (AngeloCG97) [inline webapp/src/components/GeoMap/CountryMap.js:101]: We delete inline style

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by adriexnet — APPROVED** [review]

**Pull request merged**